node/src/remote/index.ts

```diff
@@ -38,6 +38,9 @@ export class RemoteConnection implements Connection {
     if (!opts.uri.startsWith('db://')) {
       throw new Error(`Invalid remote DB URI: ${opts.uri}`)
     }
+    if (opts.apiKey == null || opts.apiKey === '') {
+      opts = Object.assign({}, opts, { apiKey: process.env.LANCEDB_API_KEY })
+    }
     if (opts.apiKey === undefined || opts.region === undefined) {
       throw new Error('API key and region are not supported for remote connections')
```

Comment:
Is this the right error message?

Reply:
can we also add something like `you can set api through LANCEDB_API_KEY env variable or remote db url`?

Reply:
updated